cmd/notary-server/config.go

```diff
@@ -278,3 +282,31 @@ func parseServerConfig(configFilePath string, hRegister healthRegister) (context
 		ConsistentCacheControlConfig: consistentCache,
 	}, nil
 }
+
+func setupSignalTrap() {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, notary.NotarySupportedSignals...)
+	go func() {
+		for {
+			signalHandle(<-c)
+		}
+	}()
+}
+
+// signalHandle will increase/descrease the logging level via the signal we get.
+func signalHandle(sig os.Signal) {
+	switch sig {
+	case syscall.SIGUSR1:
+		if err := utils.AdjustLogLevel(true); err != nil {
+			fmt.Printf("Attempt to increase log level failed, will remain at %s level, error: %s", logrus.GetLevel(), err)
```

Comment:
Ah ok.  Hm... I guess we have a bunch of other things (like GRPC connection failures) which may not respect the JSON logging.  Would just be nice to get these to log in the same format as specified by the `-logf` flag, but we can maybe think of something else.